CONTRIBUTING.md

```diff
@@ -23,7 +40,7 @@
 
 ```bash
 # rake tasks
-rake         # Run linters and feature tests
+rake         # Run linters and feature tests in parallel
```

Comment:
This sounds like its running the linters and feature tests in parallel which its not. I'd just remove this